index.html

```diff
@@ -198,7 +198,7 @@ <h3>Common Voice</h3>
 				</a>
 				<a class="col-sm-6 col-md-4 p-4" href="https://chaoss.community/metrics/" target="_blank" rel="noopener">
 					<img alt="" src="img/projects/chaoss.jpg" />
-					<h3>CHAOSS D&I Workgroup</h3>
+					<h3>CHAOSS D Workgroup</h3>
```

Comment:
It is D&I, could you please change the D to D&I?

Reply:
@ramyaparimi sure

Reply:
@ramyaparimi changes are done, I am sorry for replying this late.